teuthology/openstack/__init__.py

```diff
@@ -923,40 +933,43 @@ def get_user_data(self):
         log.debug("get_user_data: " + content + " written to " + path)
         return path
 
+    def server_group(self):
+        return "%s" % self.args.name
+
+    def worker_group(self):
+        return  "%s-worker" % self.args.name
+
```

Comment:
Can we add a "teuthology-" namespace prefix to the `server_group` and `worker_group` values, please? 

The reason why this is preferable: without a namespace, a user might run t-o with `--name foo` in an environment with an existing `foo` security group that is unrelated to teuthology.

Reply:
The same applies to the teuthology and packages-repository VMs themselves - if `--name foo` is given, then the teuthology VM should not be called "foo", because there might already be a VM with that name. Of course, openstack will still create the new VM "foo" but it's very confusing for users looking at the list of instances in the region when they see two instances both called "foo".

Reply:
If we add everywhere prefix like 'teuth-' this will not allow to teardown existing systems. Also, you will not be possible to give server name exactly which you like because it will always have 'teuth-' prefix. As a convention one can always give `--name=teuthology-foo` instead `--name=foo`.

Reply:
> If we add everywhere prefix like teuth- this will not allow to teardown existing systems.

By this, you mean systems running from before this PR is merged? I don't think that's particularly important - those can be deleted manually.

Reply:
> you will not be possible to give server name exactly which you like because it will always have teuth- prefix.

Always having the prefix is the whole point. We cannot rely on users to enter the prefix by themselves.

Keep in mind that the tenant/region may be used by multiple people and not all of them are necessarily using teuthology.

With the current master, it is always clear which VMs belong to the teuthology environment. But if we merge this PR as it is, this will no longer be the case. We will suddenly start having teuthology VMs named "foo", "bar", etc. This would be a regression because it would no longer be possible, simply by inspecting the name, to tell what the VM is doing.